### PR TITLE
Use readFile and allows asset files to be configured

### PR DESCRIPTION
**Pull request by dawnerd**

### CODE REVIEW

File: helpers/asset.js [hunk 1]
```diff
@@ -1,16 +1,52 @@
...
-  const settings = server.settings.app;
+module.exports = function(server, asset, done) {
+  const settings = this.options.assets || {};
+  const dist = settings.dist || '';
+  const mappingFile = settings.mappingFile;
+  const endpoint = settings.endpoint || '';
+  const realm = server.realm.plugins['hapi-nunjucks-helpers'];
 
-  const dist = settings.assets.dist || '';
+  if (process.env.NODE_ENV === 'prod') {
```
Comment (jgallen23) [inline helpers/asset.js]: should this be !=?

Comment (jgallen23) [reply helpers/asset.js]: and it's `production` either way

Comment (jgallen23) [reply helpers/asset.js]: also, maybe we have an option that is passed in to cache or not

**Review by jgallen23 — CHANGES_REQUESTED** [review]

File: helpers/asset.js (hunk 1 above)
Comment (dawnerd) [reply helpers/asset.js]: Ah good catch! I'll change it to be an option

**Pull request merged**